analyses/focal-cn-file-preparation/run-prepare-cn.sh

```diff
@@ -1,12 +1,14 @@
-# Chante Bethell for CCDL 2019
-# Run 01-prepare-cn-file.R
+# C. Bethell and C. Savonen for CCDL 2019
+# Run 00-add-ploidy-cnvkit.Rmd, 01-prepare-cn-file.R, and 02-rna-expression-validation.R
+# sequentially.
 #
 # Usage: bash run-prepare-cn.sh
 
 set -e
 set -o pipefail
 
 XYFLAG=${OPENPBTA_XY:-1}
+cds_file=../../scratch/gencode.v27.primary_assembly.annotation.bed
```

Comment:
I've been meaning to do this where I have this in `snv-callers` but we should probably include an indicator in this file name that tells people it's been filtered to CDS sequences only. Because it is a scratch file and isn't meant to stick around, I haven't made it a priority to change this. My bad.